src/cloud/reference/ece-tools-reference.md

```diff
@@ -20,30 +20,35 @@ php ./vendor/bin/ece-tools list
 ```terminal
 Available commands:
   build                    Builds application
-  db-dump                  Creates backup of database
+  db-dump                  Creates backups of databases
   deploy                   Deploys application
   help                     Displays help for a command
   list                     Lists commands
   patch                    Applies custom patches
-  post-deploy              Performs after deploy operations.
+  post-deploy              Performs after deploy operations
   run                      Execute scenario(s)
  backup
   backup:list              Shows the list of backup files
   backup:restore           Restore important configuration files. Run backup:list to show the list of backup files
  build
   build:generate           Generates all necessary files for build stage
   build:transfer           Transfer generated files into init directory
+ cloud
+  cloud:config:create      Create .magento.env.yaml file from provided configuration. Overwrite file if it exists
+  cloud:config:update      Update .magento.env.yaml file with provided configuration. Create file if it does not exist
```

Comment:
```suggestion
  cloud:config:create       Creates a `.magento.env.yaml` file with the specified build, deploy, and post-deploy variable configuration. Overwrites any existing `.magento,.env.yaml`file.
  cloud:config:update      Update the existing `.magento.env.yaml` file with the specified configuration. Create `.magento.env.yaml` file if it does not exist.
```

Reply:
This is a little painful, but I'd suggest adding "." to all the table descriptions.  Because some entries have multiple sentences, it will be inconsistent if all entries do not have periods.  In most cases, the description is a sentence.

Reply:
Updated